apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/BehandlingGrunnlagRoutes.kt

```diff
@@ -81,5 +86,18 @@ fun Route.behandlingGrunnlagRoute(
                 }
             }
         }
+
+        post("oppdater-grunnlag") {
```

Comment:
Når skal det her brukes?
For du henter ting ut av grunnlag for å så legge det inn igjen som en andre opplysningstyper, er det et annet sted vi gjør noe feil?

Reply:
Det er tiltenkt situasjoner der søknaden var ugyldig i henhold til PDL ved mottak av søknad. Kanskje PDL var utdatert, så blir den oppdatert og da kan grunnlaget oppdateres av SB slik at man kan gå videre med søknaden.